client/signup/config/get-onboarding-url.js

```diff
@@ -0,0 +1,14 @@
+import config from '@automattic/calypso-config';
```

Comment:
Considering that this helper function is used all over the place (`/me`, sites, inline help and so on) is this a good place for this function? I'd consider placing it somewhere in `/lib` instead.

Reply:
good point! I had a brain fart and thought that `/signup` was inside of `/lib`. I found the `/lib/paths/` folder which has helper functions for generating paths such as for login and the editor. I think this is a good place for it, what do you think?

In moving it there I found that the naming convention is not to use the `get` prefix. I think `get` was used because previously the function was a selector from the state. I renamed the function to just `onbaordingUrl()`.

Reply:
Sounds great, thank you 👍